charts/loki-stack/values.yaml

```diff
@@ -13,7 +13,7 @@ grafana:
     datasources:
       enabled: true
   image:
-    tag: 8.3.4
+    tag: 8.3.5
```

Comment:
@thojkooi 
Could you separate this PR please?

Reply:
I've moved this to #1028